corehq/tabs/tabclasses.py

```diff
@@ -756,6 +757,16 @@ def _get_export_data_views(self):
                         ] if _f]
                     })
 
+            if toggles.EXPORT_DATA_SOURCE_DATA and self.can_view_case_exports and self.can_view_form_exports:
```

Comment:
Two things come to mind:
1. what if we have a `self.can_view_datasource_exports` function and move `self.can_view_case_exports and self.can_view_form_exports` into that? Feel free to dispute / disregard this. It's just a feeling I have.
2. Do we maybe want a separate datasource view permission? I'm not 100% what the reasoning behind the separate case and export permissions are, but should we follow that?  :thinking:

Reply:
> The reasoning behind having form and case export permission is that a datasource is essentially form/case data, hence exporting the datasource is akin to exporting form/case data.

Yeah I was thinking as much. Seems fine then